rebar.config

```diff
@@ -28,7 +28,7 @@
         {eper, ".*", {git, "git://github.com/basho/eper.git", {tag, "0.78"}}},
         {sext, ".*", {git, "git://github.com/basho/sext.git", {tag, "1.1p3"}}},
         {riak_pipe, ".*", {git, "git://github.com/basho/riak_pipe.git", {branch, "2.0"}}},
-        {riak_api, ".*", {git, "git://github.com/basho/riak_api.git", {branch, "2.0"}}},
-        {riak_dt, ".*", {git, "git://github.com/basho/riak_dt.git", {branch, "2.0"}}},
+        {riak_api, ".*", {git, "git://github.com/basho/riak_api.git", {branch, "feature/gsets"}}},
```

Comment:
@fadushin maybe we can just focus and give @binarytemple-bet365 info how which branch to make this PR against too?